packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -43,14 +58,15 @@ export default class RatingBar extends PureComponent {
       </div>
     )
   }
-
+  
   render() {
     const {
       theme,
       label,
       className,
       score,
       maxScore,
+      ratingValue,
```

Comment:
All these props could be confusing (what is `ratingValue`, what is `score`, they seem to imply the same thing). Maybe call this `prefixLabel` or something like that.